Project.toml

```diff
@@ -1,10 +1,16 @@
 name = "Match"
 uuid = "7eb4fadd-790c-5f42-8a69-bfa0b872bfbf"
-version = "1.2.0"
-author = "Kevin Squire <kevin.squire@gmail.com>"
+version = "2.0.0"
+authors = ["Neal Gafter <neal.gafter@relational.ai>"]
```

Comment:
keep kevin as an author?

Reply:
Also, he doesn't want to be contacted about this.